### ISSUE TITLE
Input language is Japanese when renaming bubbles.

### ISSUE THREAD

**Issue by Uzukii** [ONElua/ONEMenu-for-PSVita#60]
I'm trying to rename my bubbles the easiest way possible by using this app and a keyboard via bluetooth, but it keeps typing in Japanese. But when I exit the app and turn off the keyboard it stays English.

**Comment by ONElua** [comment]
This happens only when tipyng? Or even the texts language on Onemenu switch to japanese?
Wich game are you trying to rename?

**Comment by Uzukii** [comment]
It only happens when typing with a keyboard. I connected a bluetooth keyboard to my PSTV to try renaming some games and the first one I tried (Malicious Rebirth) would only let me type in Japanese. I don't remember if other games did it too but I didn't have this problem with using regular controller input to type.

**Comment by DavisDev** [comment]
That's weird, and it does not seem to be the cause of the application.
Do you have the type of keyboard configured in the language you want?
This is done from settings -> devices -> external keyboard -> type -> you select your language for the keyboard.

**Comment by Uzukii** [comment]
Yeah, everything is set to English in my settings. I made a video of it happening.
https://www.sendspace.com/file/kopy5i
From start to finish you'll see me:
1. Go into settings and connect the keyboard.
2. Open OneMenu and use the controller for bubble editing. It stays English.
3. Start typing with the keyboard and it instantly switches input to Japanese.
4. Close out of OneMenu and go to language settings, which is still set to English.
5. Go back into OneMenu and use the controller to edit a bubble, which is now typing in Japanese unlike before.
6. I disconnect the keyboard, go back in OneMenu and use the controller, still typing in Japanese, without keyboard connected.
7. I close OneMenu, edit the folder name just to be typing into something else so it goes back to English.
8. I go back into OneMenu, typing with the controller in English again, connect the keyboard midway and start typing with that, then it switches to Japanese again.

So yeah, it's weird. Don't know if it's just my keyboard but it's the only bluetooth one I have.

**Comment by DavisDev** [comment]
I see, I watch the video with great curiosity, but you never reached the section indicated in the previous comment (You must have something similar to the attached capture).
I'm sure it's some simple detail of conf that is lost there.
![capture](https://user-images.githubusercontent.com/9156640/44879342-04ff7d00-ac6f-11e8-95db-9fd2c80d4f95.png)

**Comment by Uzukii** [comment]
Whoops, I went to language settings rather than 'keyboard' language settings. =/
Well, here's the correct path. Same situation.
https://www.sendspace.com/file/owooya

**Comment by DavisDev** [comment]
This situation only appears in ONEmenu?,
have you tried the keyboard in vitashell for example?

**Comment by Uzukii** [comment]
Just tried it in VitaShell to create a folder and it didn't do it. So it only seems to happen in OneMenu.

**Comment by ONElua** [comment]
Every time you connect/disconnect external keyboard
If you do not write in the language you want, please try to set the keyboard you want...
![indice](https://user-images.githubusercontent.com/25811032/45001275-64240100-af91-11e8-9a37-67cf4822e600.jpg)

**Comment by Uzukii** [comment]
^ Welp, I tried that and it just switches back to Japanese as soon as I use the keyboard.
https://www.sendspace.com/file/cwbcu4

I'm not really worried about this anymore. I went ahead and manually renamed all my games with the controller already. But if my vita's database ever gets screwed up or I put in a new memory card I'll have to do it all over again.

BTW, how come this app can't rename PS1 Classics? It doesn't detect them, only vita games and apps. Is it because those aren't in the ux0:app/ folder?
